tsconfig.json

```diff
@@ -9,6 +9,7 @@
     "sourceMap": true,
     "declaration": true,
     "strictFunctionTypes": false,
+    "esModuleInterop": true,
```

Comment:
Added this because `safe-decode-uri-component` is written in CommonJS. With this flag `safe-decode-uri-component` can be imported by:
```javascript
import safeDecodeURIComponent from 'safe-decode-uri-component'
```
rather than
```javascript
import * as safeDecodeURIComponent from 'safe-decode-uri-component'
```